client/MAVN.Service.PaymentManagement.Client/IPaymentManagementApi.cs

```diff
@@ -45,5 +45,11 @@ public interface IPaymentManagementApi
         /// <param name="request">Validate payment request</param>
         [Post("/api/payments/validation")]
         Task<string> ValidatePaymentAsync([Body] PaymentValidationRequest request);
+
+        /// <summary>
+        /// Get payment info by external id
+        /// </summary>
+        [Get("/api/payments/{externalPaymentEntityId}")]
+        Task<PaymentInfoResponse> GetPaymentInfoAsync(string externalPaymentEntityId);
```

Comment:
Let's make a request class here, so we can add other options if required later beside externalPaymentEntityId